src/mbgl/mtl/render_pass.cpp

```diff
@@ -72,6 +72,22 @@ void RenderPass::endEncoding() {
     }
 }
 
+// Reset the states and bindings of the render pass to deal
```

Comment:
```suggestion
// Resets the states and bindings of the render pass to deal
```

Maybe move this to the header and use a triple-slash comment?

Reply:
Done